content/run-nym-nodes/troubleshooting.md

```diff
@@ -329,6 +329,11 @@ bob@nym:~$ tree /home/nym/.nym/mixnodes/
 If you `cat` the `public_sphinx.pem key`, the output will be different from the public key you will see on Nym [dashboard](https://testnet-finney-explorer.nymtech.net/). The reason for this is that `.pem` files are encoded in **base64**, however on the web they are in **base58**. Don't be confused if your keys look different. They are the same keys, just with different encoding :).
 {{% /notice %}}
 
+## What is `verloc` and do I have to configure my mixnode to implement it?
+`verloc` is short for _verifiable location_. Mixnodes and gateways now measure speed-of-light distances to each other, in an attempt to verify how far apart they are. In later releases, this will allow us to algorithmically verify node locations in a non-fakeable and trustworthy manner.
+
+You don't have to do any additional configuration for your node to implement this, it is a passive process that runs in the background of the mixnet from version `0.10.1` onwards. _Measurement packets_ encrypted as standard sphinx packets will be sent through the network and your mixnode will process them as it would any other sphinx packet it receives.
```

Comment:
One comment here - verloc sends TCP packets between nodes to measure rtt. These are *not* Sphinx packets, they're just sequence identifiers if I remember correctly. I think just remove the final sentence (`Measurement packets` to `receives`. I'll be in meetings, please merge this and deploy when appropriate :).

Reply:
resolved in 0386b5c151b35626e474831c65eebccd864510cb